apps/e2e-tests/configuration.ts

```diff
@@ -0,0 +1,16 @@
+import appUrls from "./urls/app-urls";
+import saleorUrls from "./urls/saleor-urls";
+
+export const appName = process.env.APP_NAME;  // TODO: name should be taken from the manifest to eliminate possible footguns
+export const appUrl = process.env.APP_URL;
+export const instanceUrl = process.env.INSTANCE_URL;
+export const dashboardUserEmail = process.env.DASHBOARD_USER_EMAIL;
+export const dashboardUserPassword = process.env.DASHBOARD_USER_PASSWORD;
```

Comment:
how are we going to control it? this assumes we configure it per env, but how we will orchestrate it?

maybe credentials and instance should be in ENV, but app name & URL can be some static config?
we can create a json-like file (maybe even automatically generate it from fs) to create a list of apps to run tests for all of them in parallel?

I guess we'd like to run smokes for every app eg when we bump app-sdk for all of them